src/Sulu/Component/Media/Tests/Unit/SystemCollections/SystemCollectionManagerTest.php

```diff
@@ -345,17 +345,29 @@ public function testWarmUpNotFresh($config, $existingCollections, $notExistingCo
         $collectionManager = $this->getCollectionManager($existingCollections, $notExistingCollections);
 
         $entityManager = $this->prophesize(EntityManagerInterface::class);
-        $cache = $this->prophesize(CacheInterface::class);
-        $cache->isFresh()->shouldBeCalled()->willReturn(false);
-        $cache->write($data)->shouldBeCalled();
-        $cache->read()->shouldBeCalled()->willReturn($data);
+        $cache = $this->createMock(CacheInterface::class);
```

Comment:
use `prophesize` to mock the cache we should be consistent

Reply:
Ok, I will change but why use prophesize when is not necessary ?

Reply:
To be consistent in the code.